contrib/mpi-proxy-split/lower-half/gethostbyname-static/gethostbyname_static.c

```diff
@@ -166,12 +177,7 @@ int getaddrinfo(const char *__restrict node,
     close(pipefdin[0]);
     dup2(pipefdout[1], 1);
     close(pipefdout[1]);
-    char *argv[] = {"getaddrinfo", (char *)node, (char *)service, NULL};
```

Comment:
i think we should leave this line in, and pass `*argv` to `execvpProxyWithArgv` rather than `service`, as i'm experiencing a bug when `node` isn't passed in.